index/scorch/event.go

```diff
@@ -22,7 +22,7 @@ var RegistryAsyncErrorCallbacks = map[string]func(error, string){}
 
 // RegistryEventCallbacks should be treated as read-only after
 // process init()'ialization.
-var RegistryEventCallbacks = map[string]func(Event){}
+var RegistryEventCallbacks = map[string]func(Event) bool{}
```

Comment:
Would you add some commentary above to explain when the expected return values must be true for the function above.

Reply:
The return values are determined by the callbacks. Without a callback, they will fall back to true.